main/src/ca/uwaterloo/flix/api/lsp/HoverResult.scala

```diff
@@ -0,0 +1,34 @@
+package ca.uwaterloo.flix.api.lsp
```

Comment:
-> Add copyright header for all new files.

Reply:
(But I don't think we should have this abstraction. I think it is a "false abstraction".)

Instead, we should just use Option or Result